### DISCUSSION TITLE
I can't install stability matrix

### DISCUSSION THREAD

**Discussion by amj1984** [LykosAI/StabilityMatrix#89] [General]
I just run stability matrix then selecd the folder I want to install then it crash. and after that every time I run stability matrix give me this error for a few sec then close itself
![image](https://github.com/LykosAI/StabilityMatrix/assets/143294927/93c41f2c-9f9a-4924-856b-f29a80984ff8)

**Comment by ionite34** [comment]
Hi I think we found this issue occurs if the data directory you selected before no longer exists. Will push a fix soon but for now you can try deleting the `%APPDATA%/StabilityMatrix/library.json` file and launching the app again.

**Comment by ionite34** [comment]
Fixed in https://github.com/LykosAI/StabilityMatrix/releases/tag/v2.3.3

**Comment by mk230580** [comment]
I'm encountering an installation issue on Windows 11. Here's a screenshot. Can you assist us in resolving this?

![image](https://github.com/LykosAI/StabilityMatrix/assets/1321028/c3e24185-6139-4faa-8b00-5c71066914d4)

**Comment by viewlessfinder** [comment]
Error while installing stability matrix on windows 11. Any help would be greatly appreciated

![Screenshot 2024-01-13 000510](https://github.com/LykosAI/StabilityMatrix/assets/105215651/acc35f37-381c-4a79-9ff3-13bf82902bae)
